### PR TITLE
Fix imports and multiple workers tests

### PR DESCRIPTION
**Pull request by UnDarkle**

### CODE REVIEW

File: tests/core/multiprocess/utils.py [hunk 1]
```diff
@@ -81,13 +81,22 @@ def create_sub_process(self):
...
     resources = (request('res1', DemoResource, ip_address=IP_ADDRESS1),)
 
+    wait_for_others = 0
+    WAIT_AFTER_CHECK = 0.2
+
     def test_method(self):
         """Register the resource's ID to the shared queue."""
         self.register_id(id(self.res1))
-        time.sleep(0.5)  # Make sure the case won't be taken by the same worker
+        while self.pid_queue.qsize() < self.wait_for_others + 1:
```
Comment (osherdp) [inline tests/core/multiprocess/utils.py]: In case of a bug, will it be an infinite loop?

Comment (osherdp) [reply tests/core/multiprocess/utils.py]: Using `waiting` should be more readable

**Comment by osherdp** [comment]
Sync with master

**Comment by UnDarkle** [comment]
fixed in another branch

**Pull request closed without merging**